locksmith/__tests__/controllers/metadataController.test.ts

```diff
@@ -1,9 +1,9 @@
 import request from 'supertest'
 import * as sigUtil from 'eth-sig-util'
 import * as ethJsUtil from 'ethereumjs-util'
-import { LockMetadata } from '../../src/models/lockMetadata'
-import { KeyMetadata } from '../../src/models/keyMetadata'
-import { addMetadata } from '../../src/operations/userMetadataOperations'
+import { LockMetadata } from '../../src/tokens/models/lockMetadata'
```

Comment:
I think we should keep the `operations` hierarchy here, but maybe nest `operations/tokens`?